src/DynamoCoreWpf/ViewModels/Core/WorkspaceViewModel.cs

```diff
@@ -1129,9 +1149,20 @@ private void DoGraphAutoLayout(object o)
                 var n = graph.FindNode(node.GUID);
                 if (n != null)
                 {
+                    double deltaX = n.X - node.X;
+                    double deltaY = n.Y - node.Y;
+
                     node.X = n.X;
                     node.Y = n.Y;
                     node.ReportPosition();
+                    node.OnNodeModified();
```

Comment:
Will this cause the entire graph to re-execute? We don't want that, do we? Is there any other way to mark the file dirty?